.setup/update_database.py

```diff
@@ -133,6 +133,7 @@
         
         # categories ordering for discussion forum
         os.system("PGPASSWORD='{}' psql --host={} --username={} --dbname={} -c 'ALTER TABLE ONLY categories_list ADD COLUMN rank int'".format(*variables))
+        os.system("PGPASSWORD='{}' psql --host={} --username={} --dbname={} -c 'ALTER TABLE ONLY categories_list ADD COLUMN color varchar DEFAULT '#000080'".format(*variables))
```

Comment:
Please test this, there is a single quote missing at the end to close off the one before "ALTER"